openfisca_country_template/variables/benefits.py

```diff
@@ -75,30 +75,32 @@ class parenting_allowance(Variable):
     value_type = float
     entity = Household
     definition_period = MONTH
-    label = ''' Allowance for low income people with children to care for.
-                Loosely based on the Australian parenting pension'''
+    label = "Allowance for low income people with children to care for."
+    documentation = '''
+    Loosely based on the Australian parenting pension.
+    '''
     reference = \
         [u"https://www.servicesaustralia.gov.au/individuals/services/centrelink/parenting-payment/who-can-get-it"]
 
     def formula(household, period, parameters):
         '''
         A person's parenting allowance depends on how many dependents they have,
-        How much they, and their partner, earn
+        how much they, and their partner, earn
         if they are single with a child under 8
         or if they are partnered with a child under 6.
         '''
-        family_income = household('household_income', period)
+        household_income = household('household_income', period)
         income_threshold = 500
-        if family_income > income_threshold:
-            return 0
-        is_single = household.nb_persons(Household.PARENT)
+        income_condition = household_income < income_threshold
+
+        is_single = household.nb_persons(Household.PARENT) == 1
         ages = household.members('age', period)
         under_8 = household.any(ages < 8)
         under_6 = household.any(ages < 6)
-        if (is_single and under_8) or under_6:
-            return parameters(period).benefits.parenting_allowance
-        else:
-            return 0
```

Comment:
> In both return statements, a unique result was returned. 
But in `def formula(household, period, parameters)` the input `household` is always a numpy array that contains one or multiple households and we want the `parenting_allowance` calculation to take every household specific situation into account => we want a `parenting_allowance` result per household. So, the returned value must be a numpy array.
I tried to give an example in this pull request description with a test on two households with different situations and different `parenting_allowance` results given by one execution of this formula. 😊